tutorials/data-science-exploration/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Exploratory queries with Google BigQuery
+description: Use BigQuery to investigate arbitrarily large datasets.
+author: jerjou
+tags: Data Science, BigQuery
+date_published: 2017-05-10
+---
+[Google BigQuery](/bigquery) is a fully-managed service for data
+exploration using an SQL syntax. Once data has been loaded into the service, it
+can be queried at interactive speeds, without having to know beforehand what
+you're looking for.
+
```

Comment:
My first big question as a reader here is how BigQuery is different from the other GCP big data services, and why it's the one that this tutorial uses instead of one of the others. I don't know if there's a good comparison already somewhere you could link to, or if a sentence or two on the subject here could make things more clear.